### PR TITLE
Make the frr generated files compatible with 8.0+

### PR DESCRIPTION
**Pull request by fedepaol**
**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
/kind cleanup
> /kind feature
> /kind design
> /kind flake
> /kind failing
> /kind documentation
> /kind regression

**What this PR does / why we need it**:

FRR 8 preferred format for bfd echo-intervals is with separate transmit
/ receive timers. FRR 9.1 actually reports an error when deleting the
echo interval stanza.

Here we:
- generate an frr 8.0+ compatible frr configuration file
- declare the format of the configuration file to be 8.0 compatible

**Special notes for your reviewer**:

**Release note**:

```release-note
Make the generated frr file compatible with frr 8+: split bfd echo interval in echo tx / echo rx.
```

### CODE REVIEW

File: internal/frr/templates/bfdprofile.tmpl [hunk 1]
```diff
@@ -13,12 +13,13 @@
     echo-mode
     {{end -}}
     {{ if .profile.EchoInterval -}}
-    echo-interval {{.profile.EchoInterval}}
+    echo transmit-interval {{.profile.EchoInterval}}
+    echo receive-interval {{.profile.EchoInterval}}
```
Comment (karampok) [inline internal/frr/templates/bfdprofile.tmpl:17]: This should ideally be different commit?

**Review by karampok — APPROVED** [review]

File: internal/frr/templates/bfdprofile.tmpl (hunk 1 above)
Comment (fedepaol) [reply internal/frr/templates/bfdprofile.tmpl:17]: nope, it's actually the core of the commit "make the generated files compatible to 8.0+".
From 8.0 frr format has separate echo-intervals, to the point that frr 9+ is throwing an error when deleting echo-interval (the ci failure I got in metallb).

Let me modify the description of the commit to make it clearer.

**Review by karampok — APPROVED** [review]
